src/wallet/rpcdump.cpp

```diff
@@ -601,7 +609,7 @@ Value z_importkey(const Array& params, bool fHelp)
 
         // We want to scan for transactions and notes
         if (fRescan) {
-            pwalletMain->ScanForWalletTransactions(chainActive.Genesis(), true);
+            pwalletMain->ScanForWalletTransactions(chainActive[nRescanHeight], true);
```

Comment:
It's not really a no-op per se.  There are side effects such as the wallet getting locked and ShowProgress getting invoked.  Would be safer for z_importkey to return an RPC error if out of range and ScanForWalletTransactions to immediately check for null when invoked.